env/env_posix.cc

```diff
@@ -899,7 +272,8 @@ class PosixEnv : public Env {
     }
     if (f == nullptr) {
       result->reset();
-      return IOError("when fopen a file for new logger", fname, errno);
+      // return IOError("when fopen a file for new logger", fname, errno);
```

Comment:
Should we include error message, or clean up the comment?

Reply:
I'll change it. We should include the error message.